### PR TITLE
Make wierd glass cage into a actual boxing ring.

### PR DESCRIPTION
**Pull request by H0lySquirr3l**
image

It bugged me that the "boxing ring" on the holo suit used this weird tile glass cage instead of an actual boxing ring. So I made one! :D

### CODE REVIEW

File: code/game/objects/structures/fitness.dm [hunk 1]
```diff
@@ -63,3 +63,83 @@
 		else
 			to_chat(user, "<span class='notice'>Against your previous judgement, perhaps working out is not for you.</span>")
 			being_used = 0
+
+/obj/structure/fitness/boxing_ropes
```
Comment (Novacat) [inline code/game/objects/structures/fitness.dm:67]: This needs to be moved to a _vr file, or pushed up to Polaris

**Comment by KasparoVy** [comment]
Looks like the essence of suplex

File: code/game/objects/structures/fitness.dm (hunk 1 above)
Comment (H0lySquirr3l) [reply code/game/objects/structures/fitness.dm:67]: I can do that. But how do I change the map merger? I should have had the most recent version as of this afternoon.

**Comment by H0lySquirr3l** [comment]
Ok should be fixed...

**Pull request closed without merging**